pkgs/development/libraries/gobject-introspection/default.nix

```diff
@@ -40,6 +40,7 @@ stdenv.mkDerivation rec {
   setupHook = ./setup-hook.sh;
 
   patches = [
+    ./macos-shared-library.patch
```

Comment:
Could you name the patch something like `macos-absolutize-relocatable-install-name` and add a coment with a brief description here?